### DISCUSSION TITLE
openvpn

### DISCUSSION THREAD

**Discussion by kevinliubin** [Homebrew/discussions#3201] [Getting started]
I can't download OpenVPN through the command (brew install OpenVPN), because there is no 1.28.0 file in bz2 format

Download the address github provide is https://github.com/OpenSC/pkcs11-helper/releases/download/pkcs11-helper-1.28.0/pkcs11-helper-1.28.0.tar.bz2 But brew install OpenVPN The address is https://github.com/OpenSC/pkcs11-helper/releases/download/pkcs11-helper-1.28/pkcs11-helper-1.28.0.tar.bz2

**Comment by carlocab** [comment]
Should be fixed now. Do `brew update` and try again.
